pywr/nodes.py

```diff
@@ -746,4 +746,86 @@ def __init__(self, model, name, nodes, **kwargs):
         super(AggregatedNode, self).__init__(model, name, **kwargs)
         self.nodes = nodes
 
+class Transfer(Node):
```

Comment:
I think this is a good implementation. My only comment is whether to stick with the name `Transfer`. This is a quite generic extenstion to the basic `Link`. Should it derive its name from that instead?